### PR TITLE
Replace multi-line comment with single line comments

### PR DESCRIPTION
**Pull request by nick11703**
When using Compass to compile the sass files, mulit-line comments are
not stripped in the output, causing malformed CSS output.

Example
```css
.video-js {
  /* display:inline-block would be closer to the video el's display:inline
   * but it results in flash reloading when going into fullscreen [#2205]
   */
  display: block;
  /* Make video.js videos align top when next to video elements */
  vertical-align: top;
  box-sizing: border-box;
  color: #fff;
  background-color: #000;
  position: relative;
  padding: 0;
  /* Start with 10px for base font size so other dimensions can be em based and
  easily calculable. */
  font-size: 10px;
  line-height: 1;
  /* Provide some basic defaults for fonts */
  font-weight: normal;
  font-style: normal;
  /* Avoiding helvetica: issue #376 */
  font-family: Arial, Helvetica, sans-serif;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  /* Fix for Firefox 9 fullscreen (only if it is enabled). Not needed when
  checking fullScreenEnabled. */
}
```

Chrome's interpretation of the CSS
screen shot 2015-11-19 at 12 47 58 pm

### CODE REVIEW

**Comment by gkatsev** [comment]
I'm okay with changing the single line comments to use `//` instead of `/* */`. I'm not a fan of changing multi-line comments to use single-line comments. Seems like this is a bug that compass should fix instead of us fixing our code.

**Comment by nick11703** [comment]
Compass will ignore stripping multi-line comments and output them in the final CSS. The problem really come from [multi-line comments in CSS class definitions.](https://github.com/videojs/video.js/blob/master/src/css/components/_layout.scss#L2) Single line comments are stripped by compass during compilation. Can we agree to replace multi-line comments inside CSS definitions with single line comments? If so, I can update the pull request.

**Comment by gkatsev** [comment]
Can you open an issue against compass regarding this? They should strip out comments that won't be valid in pure css. I'll pull it in then with the change for those.
Sounds good?

**Comment by misteroneill** [comment]
It isn't Compass doing it, it's Sass. I don't think it's a bug in Sass, either. It may be a Chrome bug because it sees that commented out CSS property as an actual CSS property.

**Comment by misteroneill** [comment]
I did a test. The following SCSS has its comment preserved, but shows no issues in the Chrome dev tools:

```scss
.video-js {
  /*
  baz baz baz baz
  bar bar bar bar
  */
  display: block;
}
```
But this shows the behavior in the attached screenshot:

```scss
.video-js {
  /*
  display: inline
  baz baz baz baz
  bar bar bar bar
  */
  display: block;
}
```

**Comment by gkatsev** [comment]
Yeah, I just looked at videojs's css and the comments are preserved. Though, it's still working fine.

**Comment by nick11703** [comment]
I don't believe it's a Compass issue either. Sass states it will ignore multi-line comments. I wonder if it's chrome picking up the ```display:inline-block``` in the comment.

**Comment by nick11703** [comment]
Either way, if it's a Sass, Compass or browser issue, is it necessary to output the comments in the compiled CSS. They remain in the .scss files for the developer. To help make the web faster it would reduce final output by allowing single line comments to be stripped.

**Comment by misteroneill** [comment]
FWIW, the comments do get stripped in `dist/video-js.min.css`, which is really what people should be using.

**Comment by gkatsev** [comment]
I think I know what the exactly issue here.
Since comments aren't valid inside blocks, chrome looks at the block line by line to see whether it's valid or not and so if you have a multi-line comment, some of the things that are commended out could be interpreted incorrectly.

I looked through the changes and I think so of the things can just be moved outside of the blocks. The things that should stay inside the block can be changed to single line comments. I'll go through and make comments about it.

File: src/css/components/_progress.scss [hunk 1]
```diff
@@ -21,30 +21,30 @@
...
-/* Also show the current time tooltip */
+// Also show the current time tooltip
 .video-js .vjs-progress-control:hover .vjs-mouse-display:after,
 .video-js .vjs-progress-control:hover .vjs-play-progress:after {
   display: block;
 
-  /* If we let the font size grow as much as everything else, the current time tooltip ends up
-     ginormous. If you'd like to enable the current time tooltip all the time, this should be disabled
-     to avoid a weird hitch when you roll off the hover. */
+  // If we let the font size grow as much as everything else, the current time tooltip ends up
```
Comment (gkatsev) [inline src/css/components/_progress.scss]: move this out of the block

File: src/css/components/_progress.scss [hunk 2]
```diff
@@ -83,7 +83,7 @@
 // Current Time "tooltip"
 .video-js .vjs-mouse-display:after,
 .video-js .vjs-play-progress:after {
-  /* By default this is hidden and only shown when hovering over the progress control */
+  // By default this is hidden and only shown when hovering over the progress control
```
Comment (gkatsev) [inline src/css/components/_progress.scss]: move out of the block

File: src/css/components/_progress.scss [hunk 3]
```diff
@@ -101,18 +101,18 @@
 }
 
 .video-js .vjs-load-progress {
```
Comment (gkatsev) [inline src/css/components/_progress.scss]: move both the comments out of the block

File: src/css/components/_progress.scss [hunk 4]
```diff
@@ -101,18 +101,18 @@
...
   background: rgba($secondary-background-color, $secondary-background-transparency);
 }
 
-/* there are child elements of the load progress bar that represent the
-specific time ranges that have been buffered */
+// there are child elements of the load progress bar that represent the
+// specific time ranges that have been buffered
 .video-js .vjs-load-progress div {
-  /* For IE8 we'll lighten the color */
+  // For IE8 we'll lighten the color
```
Comment (gkatsev) [inline src/css/components/_progress.scss]: move both the comments out of the block

**Comment by gkatsev** [comment]
@nick11703 if you make the changes discussed, I'll pull it in. Thanks!

**Comment by nick11703** [comment]
@gkatsev No problem. I'll work on that soon. Thanks.

File: src/css/components/_progress.scss (hunk 4 above)
Comment (nick11703) [inline src/css/components/_progress.scss]: Both of these comments make sense where they are as single line ```//``` comments that will get stripped. Do you really want to move these?

Comment (gkatsev) [inline src/css/components/_progress.scss]: I was thinking that this block is small and it's fairly obvious what is happening we can move it out. But yeah, I guess it's fine to leave it as is.

**Comment by gkatsev** [comment]
@nick11703 hey, did you get a chance to go through and update the comments? (Looks like this also needs a rebase).

**Comment by nick11703** [comment]
@gkatsev Sorry. Got caught up with the holidays. It's in my plans for today.

**Comment by nick11703** [comment]
@gkatsev This should be good to go now. Let me know if you need anything else.

**Comment by gkatsev** [comment]
LGTM

**Comment by dmlap** [comment]
LGTM

**Pull request closed without merging**